src/content/snapshotOptions/ignoring-elements.mdx

```diff
@@ -0,0 +1,166 @@
+---
+layout: "../../layouts/Layout.astro"
+title: Ignore tests and elements
+description: Learn how you can tell Chromatic to ignore changes for certain elements
+sidebar: { order: 4, label: Ignore elements }
+---
+
+import ParamsCallout from "../../components/ParamsCallout.astro";
+
+import IntegrationSnippets from "../../components/IntegrationSnippets.astro";
+
+# Ignore tests and elements
+
+Sometimes a component's appearance changes every render or contains content like video and [animation](/docs/animations) that is impossible to test consistently. This will trigger visual changes even when the component code hasn't changed. Ignore tests or DOM elements to tell Chromatic to skip them when looking for changes.
+
+## Ignore tests
+
+By default, Chromatic captures snapshots for all your UI components, whether you're testing with Storybook or with the E2E integration (i.e., [Playwright](/docs/playwright) or [Cypress](/docs/cypress)), ensuring your UI remains consistent at all times. However, you may want to ignore specific tests that are irrelevant or cause false positives. 
+
+### With Storybook
+
+If you're running tests with Storybook, you can enable the ` disableSnapshot` option to configure Chromatic to ignore stories and prevent them from being snapshotted. This is useful if you're adopting Chromatic incrementally and want to turn off snapshotting for specific stories or work with components that could contain dynamic content or animations that may trigger unwanted visual changes.
+
+```ts title="src/components/NotFound.stories.ts|tsx"
+// Adjust this import to match your framework (e.g., nextjs, vue3-vite)
+import type { Meta, StoryObj } from "@storybook/your-framework";
+
+/*
+ * Replace the @storybook/test package with the following if you are using a version of Storybook earlier than 8.0:
+ * import { within } from "@storybook/testing-library";
+ * import { expect } from "@storybook/jest";
+ */
+import { expect, within } from "@storybook/test";
+
+import { NotFound } from "./NotFound";
+
+const meta: Meta<typeof NotFound> = {
+  component: NotFound,
+  title: "NotFound",
+  parameters: {
+    // Disables Chromatic's snapshotting on a component level
+    chromatic: { disableSnapshot: true },
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof NotFound>;
+
+export const Default: Story = {
+  play: async ({ canvasElement }) => {
+    const canvas = within(canvasElement);
+    await expect(canvas.getByText("Page not found")).toBeInTheDocument();
+  },
+};
+```
+
+<ParamsCallout name="disableSnapshot" integration="storybook" />
+
+### With E2E
```

Comment:
We should use `Playwright & Cypress` instead of E2E

Reply:
I'm good with the update. Appreciate the suggestion